src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -270,7 +275,9 @@ export class HeaderFilter<T extends Slick.SlickData> {
         $filterButton.appendTo(args.node);
 
         this.columnFilterButtonMapping.set(column.id!, filterButton);
-        this.columnSortButtonMapping.set(column.id!, SortProperties.NONE);
+        if (this.columnSortButtonMapping.get(column.id!) === undefined) {
```

Comment:
`columnSortButtonMapping` maps from column id to the col sort state instead of sort button, right?

Reply:
yes that's correct, I can rename it

Reply:
IIRC the same also applies to the filter mapping name but you can fix it when you get a chance